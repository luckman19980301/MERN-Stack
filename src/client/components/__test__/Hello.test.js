import React from 'react';
import { shallow } from 'enzyme';
import Hello from '../Hello';

describe('Hello component', () => {
  const wrapper = shallow(<Hello />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
