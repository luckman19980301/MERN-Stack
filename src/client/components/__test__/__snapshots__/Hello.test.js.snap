// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello component renders correctly 1`] = `ShallowWrapper {}`;
